### ISSUE TITLE
Smart Plug no longer pairing to Homey

### ISSUE THREAD

**Issue by lovethyresson** [Signalare/com.se.wiserapp#50]
I can't get my Schneider Smartplug (CCT711119) to pair to Homey anymore. It works fine pairing them to the Wiser GW.

Homey 8.0.2

**Comment by h3rman** [comment]
Where do you get stuck? Does Homey find the device? Get any error?

**Comment by lovethyresson** [comment]
Yeah the device is in pairing mode and reset correctly, it does work fine connecting it to the wiser gateway.

Interestingly enough it doesn't work adding it as a "generic zigbee device" either, it just searches and eventually times out.

This kind of indicates could be a broader zigbee problem rather than specific to the Wiser app. Having said that, other zigbee devices pair correctly.

Do you know how I could troubleshoot this further?

**Comment by lovethyresson** [comment]
You mean like orphans? Nothing there from what I can see (see screenshot).
Weirdly enough, now all of a sudden none of my rotary dimmers work either. They look like they do from Homey, but nothing actually happens on the devices.. Very weird.

<img width="1500" alt="image" src="https://user-images.githubusercontent.com/57791059/179033615-2ba4ef8a-dbd8-45e1-ab1f-24b91a413931.png">

I submitted a diagnostics report: 82d29c94-72b1-419d-8236-25a46df5dfd8
